### ISSUE TITLE
Pokémon Omega Ruby/Alpha Saphire ‒ Top screen garbling bug

### ISSUE THREAD

**Issue by Dragios** [citra-emu/citra#2246]
This happens when you found a wild pokemon and it shows the transition effect before starting the battle
Using *Hardware Renderer* displays this during transition:
![Pixelated graphic](https://cloud.githubusercontent.com/assets/18496868/20775971/b7288710-b798-11e6-9b55-9c17623f7214.png)

Nothing useful in log:
```
Service.APT <Warning> core\hle\service\apt\apt.cpp:SetScreenCapPostPermission:416: (STUBBED) screen_capture_post_permission=3
Service.CFG <Warning> core\hle\service\cfg\cfg.cpp:GenHashConsoleUnique:177: (STUBBED) called app_id_salt=0x55D
Service.FRD <Warning> core\hle\service\frd\frd.cpp:GetMyFriendKey:91: (STUBBED) called
```
Switching to *Sofware Renderer* will playback the animation exactly like the actual hardware behaviour.

Tested this on commit 3174bfd

**OS:** _Windows 10 build 14393.447 (codename Redstone 1)_
**GPU:** _nVidia GT 610 2GB DDR3 (Asus)_
**Driver:** _Version 375.95_
**OpenGL:** _Version 4.5_

**Comment by Dragios** [comment]
![Subv](https://cloud.githubusercontent.com/assets/18496868/20776747/75f3ab6c-b79d-11e6-93d3-e04c70525230.png)

**Comment by Subv** [comment]
Could you provide a save file for this?

**Comment by Dragios** [comment]
[Pokémon Omega Ruby save file.zip](https://github.com/Dragios/Cloud-Drive/blob/master/Pok%C3%A9mon%20Omega%20Ruby%20save%20file.zip)

**Comment by Subv** [comment]
After some debugging, the framebuffer contains the correct image, but it gets broken as we perform the display transfer.
It appears to be a bug in our RasterizerOpenGL::AccelerateDisplayTransfer implementation, as returning false from that function (Causing the CPU to perform the transfer) makes the problem go away.

**Comment by Dragios** [comment]
Most likely related to this as it has the exact same display issue and yet that one is solved but not this one
Under **Pokemon Mystery Dundeon: Gates To Infinity** screenshot section
_**@wwylele** commented on [3 Apr](https://github.com/citra-emu/citra/pull/1436#issuecomment-204888668)_

**Comment by yuriks** [comment]
Please verify it's fixed with #2266

**Comment by Dragios** [comment]
Confirm fixed
